src/maps/map_data_importer.cpp

```diff
@@ -159,6 +159,7 @@ void maps::MapDataImporter::ImportProvinces(const commonItems::ModFilesystem& mo
             }
          }
       }
+      Log(LogLevel::Progress) << 50 + (10 * y / height);
```

Comment:
Does this have an impact on conversion speed or the size of the log? I'd kind of expect it to.

Reply:
Will optimize.